sdk/nodejs/api/test/api.spec.ts

```diff
@@ -370,6 +371,24 @@ describe("NodeJS SDK api", function () {
     )
   })
 
+  it("Handle enumeration", async function () {
+    this.timeout(60000)
+
+    await connect(async (client) => {
+      const exportID = `./export-${randomUUID()}`
+
+      const isSuccess = await client
+        .container()
+        .from("alpine:3.16.2")
+        .export(exportID, {
```

Comment:
nit: `export` is a somewhat expensive API, and this doesn't actually check that the given value was respected (i.e. it's in Docker format).

Maybe a cheaper test would be to expose a UDP port on a container, get the exposed ports, and assert that the port has the right value? It's double-testing "returning list of objects" but I can't think of a better alternative. :thinking:

Reply:
Updated in [2eb6f43](https://github.com/dagger/dagger/pull/5594/commits/2eb6f43918ef9281e21445f2c6da7c87b99ddd61)